apps/site/lib/site_web/templates/event/_event_list.html.eex

```diff
@@ -3,14 +3,23 @@
   <%= content_tag(:h2, @title, extra_classes) %>
   <ul class="list-group list-group-flush">
     <%= for event <- @events do %>
+      <%
+        range = case event do
+          %Content.Teaser{} = item -> %{start: item.date, stop: item.date_end}
+          %Content.Event{} = item -> %{start: item.start_time, stop: item.end_time}
+        end
+
+        path_fn = case event do
+          %Content.Teaser{path: path} -> fn -> cms_static_page_path(@conn, path) end
+          %Content.Event{} -> fn -> event_path(@conn, :show, event) end
+        end
+      %>
```

Comment:
I didn't read too deeply into this, but I though that the list would always be `Teaser.t()` and not `Event.t()`?

Reply:
We're keeping support for when this template i used by the very old (and not often used) Upcoming Board Meetings paragraph, which serves us `Event.t()` structs. See [this commit](https://github.com/mbta/dotcom/pull/85/commits/64c7215f9ff8225c760ed0bbb5ba80c9d4213884) for a little more detail.

Reply:
FYI: https://app.asana.com/0/385363666817452/1130867967246842